PeerioTechnologies/peerio-mobile
# feat: disable shared folder actions while converting [branch ch2764]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/2764/mobile-disabling-actions-for-in-progress-sharing-folders-shared-folders
#### Testing instructions  
2 cases:
1 - In files:
a- Tap the *local* folder action sheet while a local folder is being converted to a shared folder. All actions should be disabled.
b- Tap the *shared* folder action sheet while a local folder is being converted to a shared folder. All actions should be disabled.

2 - In chat:
a- Start the local folder conversion process from files list by sharing a folder. Make sure theres enough files in the folder so that the conversion takes some time. Go to the chat that it was shared in, tap on the folder action sheet in chat. There's an extra action button here: "unshare", regardless, all actions should be disabled.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
